custom_events/models/materials.py

```diff
@@ -0,0 +1,21 @@
+
+# # -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class Materials(models.Model):
+    _name = 'material.materials'
+    
+    name = fields.Char(string="Name" )
+    Kinds_materials = fields.Selection(
```

Comment:
@YounisBaomamn you should not use a capital case in the fields name so rename it to **type**